### PR TITLE
Do not treat 'm'(hashkey) as a regex match begin

### LINKED ISSUE #50: RegEx match break highlighting
screenshot from 2015-07-24 09 14 24
```perl
eval {
	$page ||= $self->{m}->current_comp()->path();
};
```

### PR DESCRIPTION
**Pull request by Fank**
Fixes #50

### CODE REVIEW

File: spec/grammar-spec.coffee [hunk 1]
```diff
@@ -70,6 +70,12 @@ describe "perl grammar", ->
       expect(tokens[13]).toEqual value: "/", scopes: ["source.perl", "string.regexp.find-m.simple-delimiter.perl", "punctuation.definition.string.perl"]
       expect(tokens[14]).toEqual value: "smx", scopes: ["source.perl", "string.regexp.find-m.perl", "punctuation.definition.string.perl", "keyword.control.regexp-option.perl"]
 
+    it "does not treat 'm'(hashkey) as a regex match begin", ->
+      {tokens} = grammar.tokenizeLine("$foo->{m}->bar();")
+      expect(tokens[3]).toEqual value: "{", scopes: ["source.perl"]
+      expect(tokens[4]).toEqual value: "m", scopes: ["source.perl", "constant.other.bareword.perl"]
+      expect(tokens[5]).toEqual value: "}", scopes: ["source.perl"]
+
```
Comment (50Wliu) [inline spec/grammar-spec.coffee]: :art: Extra newline here :P

**Comment by 50Wliu** [comment]
Looks good to me!

**Comment by Fank** [comment]
@50Wliu i :art:ed it :smiley:

**Comment by kevinsawicki** [comment]
Thanks for this :ship:

**Pull request merged**